site/specs-source/voice.json

```diff
@@ -2822,34 +2853,40 @@
           "startTime": {
             "nullable": true,
             "type": "string",
-            "format": "date-time"
+            "format": "date-time",
+            "description": "The time the call was initiated, in ISO 8601 format. `null` if the call is still in your queue."
           },
           "answerTime": {
             "nullable": true,
             "type": "string",
-            "format": "date-time"
+            "format": "date-time",
+            "description": "Populated once the call has been answered, with the time in ISO 8601 format."
           },
           "endTime": {
             "nullable": true,
             "type": "string",
-            "format": "date-time"
+            "format": "date-time",
+            "description": "Populated once the call has ended, with the time in ISO 8601 format."
           },
           "disconnectCause": {
             "nullable": true,
             "type": "string",
-            "description": "The reason the call was disconnected, or null if the call is still active. Current values are 'cancel', 'timeout', 'busy', 'rejected', 'hangup', 'invalid-bxml', 'callback-error', 'application-error', 'error', 'account-limit', 'node-capacity-exceeded' and 'unknown'. Additional causes may be added in the future, so your application must be tolerant of unknown values."
+            "description": "Populated once the call has ended, with the reason the call ended. See above for possible values."
```

Comment:
This is correct. But since the disconnect reasons are at the top of the `Get Call Information` section and not directly above, you could just link back to /apis/voice/#operation/getCall and the reasons would be right there.

Reply:
In the interest of time, are you okay with me ignoring this suggestion?

Reply:
sure